.github/workflows/ci.yml

```diff
@@ -65,10 +62,8 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
-      - uses: actions-rs/cargo@844f36862e911db73fe0815f00a4a2602c279505 # v1.0.3
-        with:
-          command: test
-          args: --test '*'
+      - name: Run integration-tests coverage
+        run: make integration-tests
```

Comment:
I've also took this opportunity to remove the deprecated `actions-rc/cargo` action.